pom.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, 2024 Contributors to the Eclipse Foundation
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.9</version>
+    </parent>
+
+    <groupId>jakarta.query</groupId>
+    <artifactId>jakarta.query-parent</artifactId>
+    <version>1.0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>Jakarta Query</name>
+    <description>Jakarta Query defines the syntax and semantics of the query language itself, including the type system of the language.
+    </description>
+    <url>https://projects.eclipse.org/projects/ee4j.query</url>
+
+
+    <licenses>
+        <license>
+            <name>Apache-2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git://github.com/jakartaee/query.git</connection>
+        <developerConnection>scm:git:ssh://github.com:jakartaee/query.git</developerConnection>
+        <url>https://github.com/jakartaee/query</url>
+    </scm>
+    
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/jakartaee/query/issues</url>
+    </issueManagement>
+
+    <developers>
+        <developer>
+            <name>Jakarta Data Developers</name>
+            <email>query-dev@eclipse.org</email>
+            <organization>Eclipse Jakarta Query</organization>
+            <organizationUrl>https://accounts.eclipse.org/mailing-list/query-dev</organizationUrl>
+        </developer>
+    </developers>
+
+    <properties>
+        <maven.compiler.release>17</maven.compiler.release>
+        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
With `m.c.release` set there is no point in setting `m.c.source` I think.

Reply:
I tried, but when I remove it, it complains!

Reply:
I removed `<maven.compiler.source>17</maven.compiler.source>` to get:
```
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  44.972 s
[INFO] Finished at: 2025-04-19T17:39:15+02:00
```

What is `it` that complains for you?

Reply:
My bad, I might remove more than this line.

I've tested and it worked the way you said.

Thanks.